build.sbt

```diff
@@ -988,9 +1062,16 @@ lazy val `scio-parquet`: Project = project
     }.value,
     javacOptions ++= Seq("-s", (sourceManaged.value / "main").toString),
     description := "Scio add-on for Parquet",
+    unusedCompileDependenciesFilter -= Seq(
+      // required by me.lyh:parquet-avro
+      moduleFilter("org.apache.avro", "avro-compiler")
```

Comment:
set as `provided` by `me.lyh:parquet-avro`. Even if not used here, declare as compile dep